### ISSUE TITLE
'More Actions' dropdown in Docviewer wrapping to a second line

### ISSUE THREAD

**Issue by m-brouillette** [tsgrp/HPI#1774]
As title, the 'More' actions dropdown appears on a second line and covers part of the document.  Unsure whether related to recent docviewer refactor, or simple hit the perfect edge case.

![image](https://user-images.githubusercontent.com/8784115/47295342-0ac17100-d5d5-11e8-9aff-d162a08c20b7.png)

screensize - 1920x993
`pane1` / `docViewerOutletsize` - 936x933
Trac: insuranceClaim

**Comment by m-brouillette** [comment]
@keljopap-tsgrp - Any chance this relates to the recent DocViewer refactor?

**Comment by keljopap-tsgrp** [comment]
Yes, Alex and I noted it as the main existing UI issue after the refactor but it was definitely a case of scope creep for the new tab feature.

**Comment by m-brouillette** [comment]
Still seeing this for the record.  Changing screensize can make it go back and forth : 
![image](https://user-images.githubusercontent.com/8784115/82459942-2aa16b00-9a7e-11ea-85fd-88ae3e81ad4b.png)
